### ISSUE TITLE
i do not see anything text on injector board (Picker + virtual keyboard)

### ISSUE THREAD

**Issue by KonoromiHimaries** [uBlockOrigin/uBlock-issues#55]
### Prerequisites

- [x ] I verified that this is not a filter issue
    - Filter issues MUST be reported at [filter issue tracker](https://github.com/uBlockOrigin/uAssets/issues)
    
- [x ] This is not a support issue or a question
    - Support issues and questions are handled at [/r/uBlockOrigin](https://www.reddit.com/r/uBlockOrigin/)
    
- [x ] I performed a cursory search of the issue tracker to avoid opening a duplicate issue
    - Your issue may already be reported.
- I tried to reproduce the issue when...
    - [ ] uBlock Origin is the only extension
    - [ ] uBlock Origin with default lists/settings
    - [ ] using a new, unmodified browser profile
- [x ] I am running the latest version of uBlock Origin
- [ ] I checked the [documentation](https://github.com/gorhill/uBlock/wiki) to understand that the issue I report is not a normal behavior

### Description

[Description of the bug or feature]
![2018-06-04-example domain](https://user-images.githubusercontent.com/5884000/40898353-023c0f18-67c1-11e8-9eb7-048d7a372d0d.png)

i do not see anything text on injector board if i use on-screen keyboard or click any key

### A specific URL where the issue occurs

[A specific URL is MANDATORY for issue happening on a web page, even if it happens "everywhere"]
<code>http://example.com/</code>
### Steps to Reproduce

1. [First Step] open site
2. [Second Step] open on-screen keyboard
3. [and so on...] open injector board
4. try write any filter on on-screen keyboard

### Expected behavior:

[What you expected to happen]
fix

### Actual behavior:

[What actually happened]
do not see text on injector board if i use any key

### Your environment

* uBlock Origin version: uBO 1.16.8
* Browser Name and version: Chr 67.0.3396.62
* Operating System and version: w10

**Comment by uBlock-user** [comment]
Cannot reproduce.

Did you activate the cursor on the injector board before writing ?

**Comment by KonoromiHimaries** [comment]
> Did you activate the cursor on the injector board before writing ?

how to i can simultaneously write and hold the cursor on the injector board?

**Comment by uBlock-user** [comment]
> simultaneously write on-screen keyboard and hold the cursor on the injector board?

No, just click once on the injector board after performing step 3 and then start writing via on-screen KB.

**Comment by KonoromiHimaries** [comment]
> No, just click once on the injector board after performing step 3 and then start writing via on-screen KB.

not work for me with i use the one mouse, text is still not readable

**Comment by hawkeye116477** [comment]
We can't simultaneously write with on-screen keyboard and click on cursor to highlight popup, that's not possible, cuz I must move mouse to write something. So this should be solved by adding switch like  „always highlight"

**Comment by gwarser** [comment]
"i do not see anything text on injector board" -> Text is hard  to read because of transparency/dimming. Mouse cursor must hover over panel to be clearly visible.

**Comment by uBlock-user** [comment]
@gwarser were you able to reproduce ?

**Comment by KonoromiHimaries** [comment]
I would like a more intuitive solution, how to click once in the injector board to keep it from going out until you click something else and double click on the injector board so that it does not go out until you double click this time in injector board

optionally, for a single click you can do whitelisting the application and add there, among others, an on-screen keyboard

**Comment by uBlock-user** [comment]
> how to click once in the injector board to keep it from going out until you click something else and double click on the injector board so that it does not go out until you double click this in injector board

It's by-design, the element picker will dim and become transparent when the mouse is no longer hovering, just start typing and hover again on the picker and you will see the typed text.

**Comment by gwarser** [comment]
I must type blind. On Mint, Firefox Nightly, Chromium 66 using "Onboard" keyboard. Mouse must be over interface to see text. Cannot be done with virtual keyboard. It's different for you? On Windows?

**Comment by gwarser** [comment]
Similar https://github.com/gorhill/uBlock/issues/3449 (hard to spot on first use) May be marked as accessibility issue.

**Comment by gwarser** [comment]
Longer CSS transition for opacity fading-out and somehow resetting on textarea onchange may fix this.

**Comment by uBlock-user** [comment]
If that's the issue, then it should be marked as duplicate of that.

**Comment by gwarser** [comment]
I meant for both these issues setting opacity to 1 and slowly fading out may help. For #3449 when interface is shown first time, and here on text change.

**Comment by gorhill** [comment]
This one was not fixed, not sure I can fix for it though. I will keep thinking.

**Comment by uBlock-user** [comment]
The [commit](https://github.com/gorhill/uBlock/commit/d574a09784d956b9ca77d5076ecfcc573a8f8ff4) lists it as fixed, was it not ?

**Comment by gorhill** [comment]
It was my mistake, I meant to mention the issue here as _related_, not _fixed_.

**Comment by prateekrastogi** [comment]
Despite multiple issues to fix Bing Ads, there has been no response whatsoever and agitation from repo maintainers:
So, referencing my issue here:
https://github.com/uBlockOrigin/uAssets/issues/6887#issuecomment-578509029

Let's name and shame ublock origin until their ego issues get resolved.

**Comment by krystian3w** [comment]
Try this in Chromium 105:
```css
html.paused aside:has(.CodeMirror-focused) {
    opacity: 1 ! important
}
```

But down after upgrade code mirror to v6 #2146 (maybe need days to correct rewrote) or after implemented disable #1720 (then needed checking works `textarea:focus`).
